test/exporters/psd.renderer.test.js

```diff
@@ -42,6 +42,6 @@ describe('exporters/psd', function () {
     ]
 
     let buffer = await exporterPsd.toPsdBuffer(imagesMeta)
-    assert(buffer.length === 198554)
+    assert(buffer.length === 198554 || buffer.length === 198546)
```

Comment:
I think we just care that the PSD buffer was created and has roughly the content we expect. Could just be:

```suggestion
    assert(buffer.length > 190000)
```